File: Array.py
# Create an array (list in Python)
inventory = ["Toyota", "Honda", "BMW", "Ford"]

# Add a new stock (appending to the end of the array)
inventory.append("Chevrolet")

# Delete incorrect stock (removing an item by value)
inventory.remove("Ford")

# Search for a stock (check if an item exists in the array)
stock_to_search = "Honda"
if stock_to_search in inventory:
    print(f"{stock_to_search} found in inventory.")

# Sort stocks by brand (alphabetical sorting)
inventory.sort()

# Display the inventory
print("Sorted Inventory:", inventory)

File: array.java
import java.util.ArrayList;
import java.util.Collections;

public class Inventory {
    public static void main(String[] args) {
        // Create an ArrayList (similar to a Python list)
        ArrayList<String> inventory = new ArrayList<>();
        
        // Add initial stock
        inventory.add("Toyota");
        inventory.add("Honda");
        inventory.add("BMW");
        inventory.add("Ford");
        
        // Add a new stock (appending to the end of the list)
        inventory.add("Chevrolet");
        
        // Delete incorrect stock (removing an item by value)
        inventory.remove("Ford");
        
        // Search for a stock (check if an item exists in the list)
        String stockToSearch = "Honda";
        if (inventory.contains(stockToSearch)) {
            System.out.println(stockToSearch + " found in inventory.");
        }
        
        // Sort stocks by brand (alphabetical sorting)
        Collections.sort(inventory);
        
        // Display the inventory
        System.out.println("Sorted Inventory: " + inventory);
    }
}
